msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalAuthResult.java

```diff
@@ -42,4 +46,12 @@ public MsalAuthResult(@NonNull final IAuthenticationResult authenticationResult)
     public MsalAuthResult(@NonNull final Exception exception) {
         super(exception);
     }
+
+    public static void verifyATForPop(@NonNull final String shr) throws ServiceException {
+        Map<String, ?> tokens = IDToken.parseJWT(shr);
+        // Verify if the url, path and http method are as expected
+        Assert.assertEquals("signedhttprequest.azurewebsites.net", tokens.get("u"));
+        Assert.assertEquals("/api/validateSHR", tokens.get("p"));
+        Assert.assertEquals("GET", tokens.get("m"));
```

Comment:
Let's define all the following in a constant file somewhere...since you're also using these in MsalSdk class as well:

- signedhttprequest.azurewebsites.net"
- /api/validateSHR
- GET